soc/nxp/imxrt/imxrt118x/m33/mpu_regions.c

```diff
@@ -7,13 +7,13 @@
 #include <zephyr/devicetree.h>
 #include <zephyr/arch/arm/cortex_m/arm_mpu_mem_cfg.h>
 
-#define REGION_HYPERRAM_BASE_ADDRESS   0x04000000
-#define REGION_HYPERRAM_SIZE           0x04000000
-#define REGION_DTCM_BASE_ADDRESS       0x20000000
+#define REGION_HYPERRAM_BASE_ADDRESS   0x14000000
```

Comment:
```suggestion
#define REGION_FLEXSPI2_BASE_ADDRESS   0x14000000
```
I realize you did not name these regions.  But we should not use "HYPERRAM" in the name of these regions, and should call "FLEXSPI2" instead.  This is an SOC file, to use on any board with this SOC.  Our EVK connects HyperRAM to FlexSPI2.  But other boards could use different memory options on FlexSPI2.

Reply:
Yes, you are right.